docs/cas-server-documentation/installation/Rest-Authentication.md

```diff
@@ -45,6 +45,20 @@ Expected responses from the REST endpoint are mapped to CAS as such:
 | `428`          | Produces a `AccountPasswordMustChangeException`
 | Other          | Produces a `FailedLoginException`
 
+## Warnings
+
+The remote REST endpoint can send warnings back to the CAS server using custom headers:
+
+### `X-Cas-PasswordExpirationDate`
+
+If this header is present in the response and contains a RFC1123 date a
+`PasswordExpiringWarningMessageDescriptor` is added to the created `AuthenticationHandlerExecutionResult`.
+
+### `X-Cas-Warning`
```

Comment:
Probably should be `X-CAS-etc`...

Reply:
According to https://tools.ietf.org/html/rfc7230#section-3.2 header names are case insensitive, so it should not matter, but the RFC's use Upper-Case-With-Dashes for all their headers, so I used it here too.

Reply:
It's not an RFC specific thing here; it's about correctly using a product/platform name. The proper name is "CAS".